integration/src/test/java/org/mule/test/routing/ParallelForEachTestCase.java

```diff
@@ -142,8 +141,7 @@ public void routeWithNonMuleException() {
   @Description("An error in a route results in a CompositeRoutingException containing details of exceptions.")
   public void routeWithExpressionException() {
     assertRouteException("routeWithExpressionException", EXCEPTION_MESSAGE_TITLE_PREFIX
-        + "\t1: org.mule.runtime.core.api.expression.ExpressionRuntimeException: \"Script 'invalidExpr ' has errors: \n"
-        + "\tUnable to resolve reference of invalidExpr. at 1 : 1\" evaluating expression: \"invalidExpr\".",
+        + "\t1: org.mule.runtime.core.api.expression.ExpressionRuntimeException: Script 'invalidExpr ' has errors:",
```

Comment:
did we lose the detailed error message from DW here?

Reply:
no,. they simply changed and the assesion fails because of the underlyining thing that DW does on hte failing part. This tets is just as good and not as dependant on how DW decides to format error messages